sdk/core/Azure.Core/src/Http/HttpHeader.cs

```diff
@@ -59,11 +61,15 @@ public static class Names
             public static string Range => "Range";
         }
 
+#pragma warning disable CA1034 // Nested types should not be visible
+#pragma warning disable CA1724 // Type name conflicts with standard namespace
         public static class Common
+#pragma warning restore CA1034 // Nested types should not be visible
+#pragma warning restore CA1724 // Type name conflicts with standard namespace
         {
-            static readonly string s_applicationJson = "application/json";
-            static readonly string s_applicationOctetStream = "application/octet-stream";
-            static readonly string s_applicationFormUrlEncoded = "application/x-www-form-urlencoded";
+            const string s_applicationJson = "application/json";
+            const string s_applicationOctetStream = "application/octet-stream";
+            const string s_applicationFormUrlEncoded = "application/x-www-form-urlencoded";
```

Comment:
Nit - if you have to update this PR, maybe change these to not use the `s_` prefix.